dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/GitHubSourceCodeRepo.java

```diff
@@ -908,8 +908,9 @@ private WorkflowVersion setupWorkflowFilesForGitHubVersion(Triple<String, Date,
 
         // No need to check for null, has been validated
         String primaryDescriptorPath = theWf.getPrimaryDescriptorPath();
-
         version.setWorkflowPath(primaryDescriptorPath);
+        String readmePath = theWf.getReadMePath();
```

Comment:
Really minor nit. It appears that the rest of the path variables are called `ReadMePath`, whereas here it is `readmePath`.

Probably not worth changing unless you are fixing something else at the same time.